.github/workflows/integration-tests.yml

```diff
@@ -49,7 +52,7 @@ jobs:
           - package-delete-deploy
           - sync
           - local-invoke
-          - local-start1
+          - local-start
```

Comment:
This PR is undoing all the changes that Roger sent on a recent PR:
https://github.com/aws/aws-sam-cli/pull/8399/files

Were those all wrong? I imagine maybe you just had the old version and you overrode his changes.